keyboards/handwired/twig/twig50/README.md

```diff
@@ -0,0 +1,4 @@
+Handwired 50 key with Proton C.
```

Comment:
Should follow the [template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md), and preferably be lowercase.